.dev/tests/cypress/helpers.js

```diff
@@ -115,6 +117,40 @@ export function addCoBlocksBlockToPage( clearEditor = true, blockID = '' ) {
 	cy.get( `div[data-type="coblocks/${ blockID }"]` ).should( 'exist' );
 }
 
+/**
+ * From inside the WordPress editor open the core Gutenberg editor panel
+ *
+ * @param bool clearEditor Whether or not to clear all blocks on the page before
+ * adding a new block to the page.
+ * @param clearEditor
+ * @param blockID
+ * @param string blockID     ID to check for in the DOM.
+ */
+export function addCoreBlockToPage( clearEditor = true, blockID = '' ) {
```

Comment:
We now have two similar functions for adding a block to the page. My recommendation is that we combine these into a single `addBlockToPost` and update all calls to use this new function. We should also reorder the parameters and keep our naming consistent.

**Why `addBlockToPost` vs `addBlockToPage`?**
WordPress considers all pages as a post with the `post_type` of `page`.

**Change parameter `blockID` to `blockName` and make it the first parameter**
When registering a block, the namespaced slug is called the block name so we're just keeping it consistent here so there is no potential confusion. We want to make it the first parameter because it is absolutely required to add a block to the page. We also want to remove the default `= ''` bit so that we fail if a name isn't passed. `clearEditor` isn't required so it should be last. I would also set the default value of `clearEditor` to `false` because the function name is `addBlock` not `replaceBlocksWith`.

Reply:
And to be clear with the `blockName` change, we would now be passing the full namespaced block name such as `core/image` or `coblocks/alert` to this function rather than just the last portion.

Reply:
Refactored these changes. Need to resolve a few of these tests then I'll ping you for review again.